bin/inference/pycbc_inference_plot_inj_intervals

```diff
@@ -0,0 +1,116 @@
+#! /usr/bin/env python
+""" Plots the fraction of injections with their parameter value recovered
+within a credible interval versus credible interval.
+"""
+
+import argparse
+import logging
+import matplotlib as mpl
+import matplotlib.colorbar as cbar
+import matplotlib.pyplot as plt
+import numpy
+import pycbc
+from matplotlib import cm
+from pycbc import inject
+from pycbc import transforms
+from pycbc.inference import option_utils
+
+# parse command line
+parser = argparse.ArgumentParser(usage=__file__ + " [--options]",
+                                 description=__doc__)
+parser.add_argument("--output-file", required=True, type=str,
+                    help="Path to save output plot.")
+parser.add_argument("--verbose", action="store_true",
+                    help="Allows print statements.")
+parser.add_argument("--quantiles", nargs="+", type=float,
+                    default=numpy.arange(0.05, 1.05, 0.05),
+                    help="List of quantiles to plot.")
+parser.add_argument("--injection-hdf-group", default="H1/injections",
+                    help="HDF group that contains injection values.")
+option_utils.add_inference_results_option_group(parser)
+option_utils.add_scatter_option_group(parser)
+opts = parser.parse_args()
+
+# set logging
+pycbc.init_logging(opts.verbose)
+
+# read results
+_, parameters, labels, samples = option_utils.results_from_cli(opts)
+
+# create figure
+fig = plt.figure()
+ax = fig.add_subplot(111)
+
+# typecast to list for iteratation
```

Comment:
Typo.

Reply:
Thanks spellcheck.

Fixed typo. Will probably have to rebase after #1989 is merged, and then I'll merge it.

Reply:
If you said I'm ~ spellcheck, you're welcome. :D